### PR TITLE
[MIG] website_sale_prod_description: Migration to 15.0

### PR DESCRIPTION
**Pull request by JrAdhoc**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://runbot.adhoc.com.ar/ingadhoc/website/pull/214).

File: website_sale_prod_description/views/product_template_templates.xml [hunk 1]
```diff
@@ -1,11 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo noupdate="1">
     <template id="product_description" inherit_id="website_sale.product" name="Website Sale Product Description">
-        <xpath expr="//p[@t-field='product.description_sale']" position="attributes">
-            <attribute name="t-field">product.description_website</attribute>
-        </xpath>
-        <xpath expr="//hr[@t-if='product.description_sale']" position="attributes">
-            <attribute name="t-if">product.description_website</attribute>
+        <xpath expr="//p[@t-field='product.description_sale']" position="replace">
+            <t t-if="len(product.description_website) > 1">
```
Comment (nicomacr) [inline website_sale_prod_description/views/product_template_templates.xml]:
```suggestion
            <t t-if="not is_html_empty(product.description_website)">
```

**Review by nicomacr — CHANGES_REQUESTED** [review]
@JrAdhoc Excellent Work.

Some details to fix below

**Comment by nicomacr** [comment]
@roboadhoc r+

**Comment by nicomacr** [comment]
@roboadhoc rebase-ff

**Comment by roboadhoc** [comment]
Merge method set to rebase and fast-forward

**Review by nicomacr — APPROVED** [review]

**Pull request merged**